README.md

```diff
@@ -0,0 +1,23 @@
+# VFS RAR addon for Kodi
+
+This is a [Kodi](http://kodi.tv) VFS addon to extract RAR archives or direct playback stored files in RAR archives.
+
+## Build instructions
+
+When building the addon you have to use the correct branch depending on which version of Kodi you're building against. 
+For example, if you're building the `master` branch of Kodi you should checkout the `master` branch of this repository. 
+Also make sure you follow this README from the branch in question.
+
+### Linux
+
+The following instructions assume you will have built Kodi already in the `kodi-build` directory 
+suggested by the README.
+
+1. `git clone https://github.com/xbmc/xbmc.git`
+2. `git clone https://github.com/notspiff/vfs.rar.git`
+3. `cd vfs.rar && mkdir build && cd build`
+4. `cmake -DADDONS_TO_BUILD=vfs.rar -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../../xbmc/kodi-build/addons -DPACKAGE_ZIP=1 ../../xbmc/cmake/addons`
```

Comment:
we really have to stop using those instructions. At best, they are only useful for devs.
Normal users should never build like this and only get confused.